pkg/report/cyclonedx/cyclonedx.go

```diff
@@ -363,3 +418,142 @@ func property(key, value string) cdx.Property {
 		Value: value,
 	}
 }
+
+func advisories(refs []string) *[]cdx.Advisory {
+	var advs []cdx.Advisory
+	for _, ref := range refs {
+		advs = append(advs, cdx.Advisory{
+			URL: ref,
+		})
+	}
+	return &advs
+}
+
+func cwes(cweIDs []string) *[]int {
+	var ret []int
+	if len(cweIDs) == 0 {
+		return nil
+	}
+	for _, id := range cweIDs {
+		i, err := strconv.Atoi(strings.TrimPrefix(strings.ToLower(id), "cwe-"))
+		if err != nil {
+			log.Logger.Debugf("cwe id parse error: %+v", err)
```

Comment:
Looks like `%s` is fine in this case.

```suggestion
			log.Logger.Debugf("cwe id parse error: %s", err)
```